src/services/requiredSignoffs.js

```diff
@@ -5,9 +5,26 @@ const getRequiredSignoffs = objectName =>
   axios.get(`${BASE_URL}/${objectName}`);
 const getScheduledChanges = objectName =>
   axios.get(`${BASE_URL}/scheduled_changes/${objectName}`);
+const updateRequiredSignoff = params => {
+  const { useScheduledChange, scId, ...postData } = params;
+  const type = postData.channel ? 'product' : 'permissions';
+  const scPath = scId ? `/${scId}` : '';
+  const url = useScheduledChange
+    ? `${BASE_URL}/scheduled_changes/required_signoffs/${type}${scPath}`
+    : `${BASE_URL}/required_signoffs/${type}`;
+  // TODO: where can we set this globally as a default?
+  const { accessToken } = JSON.parse(
+    localStorage.getItem('react-auth0-session')
```

Comment:
We can use the `USER_SESSION` constant instead of hard-coding `react-auth0-session`:

```jsx
import { USER_SESSION } from '../utils/constants';
```